third-party/fedora33/qemu/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# usage: build.sh qemu-version pixman-version licap_ng-version
+set -e
+
+# pixman
+tar -xzf pixman.tar.gz --skip-old-files
+cd pixman-pixman-$2
+./autogen.sh
+mkdir -p /_temp_qemu/pixman
+export CFLAGS="-fPIE"
+./configure --enable-static=yes --prefix=/_temp_qemu/pixman
```

Comment:
you could probably use the same `--prefix` for everything and then only have one entry to add to `LDFLAGS` and friends, but this is also fine :)

Reply:
Indeed, having separate prefixes was just useful for debugging build errors